docs/en/routing/basic-usage.md

```diff
@@ -0,0 +1,490 @@
+# Basic Usage
+
+## Adding Routing to an Application
+
+Begin by adding an initial route configuration, that defines a single route that maps to an identifier known as an `outlet`.
```

Comment:
I know this is structured similar in order to the I18n one. but I think it'd be better to start with registering the router first in this scenario. rather than talking about/referencing an outlet immediately

Reply:
At the moment it's:

1) Route Config
2) Registering the router
3) Using the Outlet widget

Are you suggesting the registration part before the route config? I ordered it this way because the routing config is required to register the router.

Reply:
You could switch it around to be:
* Use an Outlet in a widget
* Create a route
* Register the router

Reply:
Even re-revisiting the routes file wouldn't be bad:
* Register the router
* Use an Outlet in a widget
* Create a routes file

Reply:
I'm not sure about changing the order of these, to me it feels more natural starting with route config, adding routing to the application, then doing something that works with the registered outlet.

Reply:
when i read it i'm immediately confused by you saying put an outlet in there with no explanation what an outlet is. if you're going to keep the order, then drop the outlet part of the definition and revisit after you'd explained an outlet?

Reply:
I wish it was just `id` in the config

Reply:
Nah, that'd be confusing without any qualification in the config because an `id` is presumed unique.

Reply:
If you want to keep as is maybe just explicitly say ignoring the outlet for now, we will explore it in a later section or something along those lines.